css-overflow-5/Overview.bs

```diff
@@ -158,8 +155,42 @@ its content will be integrated into this specification,
 which will then replace it.
 Until then, this specification only contains additions and extensions to level 4.
 
-<h2 id="scroll-marker-group">
-Overflow Controls: the 'scroll-marker-group' property and pseudo-elements</h2>
+<h2 id="scroll-navigation">
+Scroll navigation controls</h2>
+
+<h3 id="scroll-markers">
+Scroll markers</h3>
+
+A <dfn>scroll marker</dfn> is any element or pseudo-element with a [=scroll target=].
+An element or pseudo-element's <dfn>scroll target</dfn> is the {{Element}} indicated by the [=scroll marker=].
+In particular, [[html#the-a-element]] and [[svg2#Links]] are scroll markers whose [=scroll target=] is the <a spec=html>indicated part</a>
+resulting from calling <a spec=html>select the indicated part</a> given the current {{Document}} and the <dfn element-attr for=a>href</dfn> attribute value as the |URL|.
+While these navigational links can be created today,
+there is little feedback to the user regarding the current content being viewed, and
+the interaction model does not match the expectations of many modern accessible UI components.
+
+This specification adds a mechanism for creating groups of [=scroll marker=]s,
+and for automatically creating '::scroll-marker'' pseudo-elements.
+Within each group, the active marker reflects the current scroll position,
+and can be styled to give the user an indication of which section they are in.
+
+Use cases include a table of contents with links to relevant contents,
+markers for scrolling carousel pages,
+and scrollable tab panels.
+
+Issue: Add images representing these examples.
+
+<h4 id="scroll-marker-grouping">
+Scroll marker grouping</h4>
+
+A <dfn>scroll marker group</dfn> of [=scroll marker=] elements is created by setting the <a href="https://open-ui.org/components/focusgroup.explainer/">focusgroup</a> attribute
+on an ancestor those [=scroll marker=]s.
+''::scroll-marker'' pseudo-elements are implicitly part of the ''::scroll-marker-group'' group created by their nearest ancestor [=scroll container=].
```

Comment:
Paragraph break here, and swap the order of the clauses maybe, like "A ::scroll-marker-group pseudo-element (described below) is the [=scroll marker group container=] for its contained ::scroll-marker pseudo-elements, which form a [=scroll marker group=] together."